app.js

```diff
@@ -127,3 +127,165 @@ message.addEventListener('change', (e) => {
   contactData.username = username.value;
   localStorage.setItem('contact-data', JSON.stringify(contactData));
 });
+
+// create popups in projects section
+
+const projects = [
+  {
+    image: 'images/work5.svg',
+    title: 'Tonic',
+    company: 'CANOPY',
+    position: 'Back End Dev',
+    year: '2015',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    skills: ['html', 'css', 'javascript'],
+    reverse: false,
+  },
+  {
+    image: 'images/work6.svg',
+    title: 'Multi-Post Stories',
+    company: 'FACEBOOK',
+    position: 'Full Stack Dev',
+    year: '2015',
+    description: `Experimental content creation feature that allows users to add to an existing story over the course
+                    of a day without
+                    spamming their friends.`,
+    skills: ['html', 'Ruby on rails', 'css', 'javascript'],
+    reverse: true,
+    rubyLink: true,
+  },
+  {
+    image: 'images/work1.svg',
+    title: 'Facebook 360',
```

Comment:
- [ ] Great job so far. Please, your mobile version `see live` and `source` buttons are not following the Figma design. Kindly add the height of your popup, so the buttons can well fit on it.
yours      
![Screenshot (875)_LI](https://user-images.githubusercontent.com/65298048/224355207-1e10f8cc-c25c-42c8-9af6-7d77835830fe.jpg))  |  
figma
![Screenshot (874)](https://user-images.githubusercontent.com/65298048/224355322-c560b77e-be0c-434b-ba73-42afeaa899f6.png)

Reply:
Fixed.